Python/aot_ceval_jit.c

```diff
@@ -4679,6 +4833,9 @@ void* jit_func(PyCodeObject* co, PyThreadState* tstate) {
 @ARM| sub sp, sp, #(num_callee_saved+num_stack_slots)*8
 @X86| sub rsp, num_stack_slots*8
 
+#if PY_MINOR_VERSION == 7
+    emit_store64_mem(Dst, arg4_idx /* =value */, sp_reg_idx, NUM_MANUAL_STACK_SLOTS * 8);
```

Comment:
A tiny improvement would be to do something like `#define WHY_SLOT NUM_MANUAL_STACK_SLOTS` so it's more clear what this line is doing.